files/system/usr/share/ublue-os/just/60-custom.just

```diff
@@ -56,10 +56,21 @@ remove-kargs-hardening:
       --delete-if-present="debugfs=off"
     echo "Hardening kargs removed."
 
-# Harden flatpaks by preloading hardened_malloc
+# Harden flatpaks by preloading hardened_malloc (highest supported hwcap)
 harden-flatpak:
     #!/usr/bin/bash
-    flatpak override --user --filesystem=host-os:ro --env=LD_PRELOAD=/var/run/host/usr/lib64/libhardened_malloc.so
+    flatpak override --user --filesystem=host-os:ro
+    #`ld-linux-x86-64.so.2 --help` prints ld.so linker info, including detected hwcaps support. Grep those, then use cut and substring selection (:0:1) to isolate the 1st character after 'v', which will be 4, 3, or 2, corresponding to the system's highest supported hwcap. On x86_64-v1 systems, grep finds no matches, leaving our variables empty.
+    uarches="$(/var/run/host/usr/lib64/ld-linux-x86-64.so.2 --help | grep '(supported, searched)' | cut -d'v' -f2)"
```

Comment:
did you test this? it's broken and has unescaped characters

Reply:
Yes, I have tested this. The unescaped ' was in the comment. Github isn't rendering the comment as a comment for some reason.

Reply:
@spaceoden sadly it looks like justfiles might not support inline comments?

Reply:
I moved the # to the front and it's rendering correctly now.